IdentityCore/src/MSIDError.m

```diff
@@ -35,6 +35,7 @@
 NSString *MSIDHomeAccountIdkey = @"MSIDHomeAccountIdkey";
 NSString *MSIDBrokerVersionKey = @"MSIDBrokerVersionKey";
 NSString *MSIDServerUnavailableStatusKey = @"MSIDServerUnavailableStatusKey";
+NSString *MSIDThrottlingCacheHitKey = @"MSIDThrottlingCacheHitKey";
```

Comment:
Do we translate this to MSAL friendly key? If yes, is there a PR for that, I missed it?

Reply:
I missed that, Here is the PR on MSAL:
https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2257